src/tests/utils/generate_jwt.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
```

Comment:
Cant we do this in python or pytest fixtures? That way we can easily pass roles to it for example.

Reply:
Let's try that idd